ggshield/cmd/iac/scan.py

```diff
@@ -157,7 +157,7 @@ def iac_scan(ctx: click.Context, directory: Path) -> Optional[IaCScanResult]:
         get_headers(
             scan_context=ScanContext(
                 command_path=ctx.command_path,
-                scan_mode="external",
+                scan_mode="directory",
```

Comment:
We have started to add an enum for the different `scan_mode` values (see core/utils.py, ScanMode enum). Can you add an `IAC_DIRECTORY = "directory"` entry to it and use it there? (the other enums should be prefixed with `SECRET_` but they are not for now).

Reply:
Done!